api/src/main/java/org/openmrs/module/eptsreports/metadata/HivMetadata.java

```diff
@@ -983,6 +983,13 @@ public Concept getBpostiveConcept() {
     return getConcept(uuid);
   }
 
+  // concept_id = 42
```

Comment:
This concept is also in TbMetadata class. Could we have have it in one place depending on use and relevance?

Reply:
[Done](https://github.com/esaude/openmrs-module-eptsreports/pull/246/commits/7a85d96ab02ad6dc0fb86669c517985486291279)